Makefile

```diff
@@ -58,3 +57,7 @@ oci-validate-json: validate.go
 oci-validate-examples: cmd/oci-validate-examples/main.go
 	go build ./cmd/oci-validate-examples
 
+%.dot.png: %.dot
```

Comment:
I like make patterns, but nothing is depending on the `media-types.dot.png` target